code/obj/submachine/seed.dm

```diff
@@ -1069,8 +1069,8 @@
 	var/can_vend = 1
 	var/seedcount = 0
 	var/maxseed = 25
-	var/category = null
 	var/list/available = list()
+	var/static/list/plant_base64_cache = list()
```

Comment:
Seems okay to me. Worth noting that if another machine wants to also show crop images similarly we should make this global rather than store one static list per machine type (probably, assuming similar use case).